src/java/htsjdk/samtools/Defaults.java

```diff
@@ -73,6 +79,7 @@
             NON_ZERO_BUFFER_SIZE = BUFFER_SIZE;
         }
         REFERENCE_FASTA = getFileProperty("reference_fasta", null);
+        REF_DOWNLOAD_ENABLED = false;
```

Comment:
Also, the property name should probably make it clear that this controls a CRAM-specific feature.